container/make-manifest-list.py

```diff
@@ -156,8 +197,57 @@ def main():
             f'v{major}.{minor}.{micro}',
             f'v{major}.{minor}.{micro}-{datetime.today().strftime("%Y%m%d")}',
         ):
-        run_command_show_failure(
-          f'podman manifest push localhost/m {base}:{t}')
+        if args.dry_run:
+            print(f'skipping podman manifest push localhost/m {base}:{t}')
+        else:
+            run_command_show_failure(
+              f'podman manifest push localhost/m {base}:{t}')
+
+def promote():
+    global args
```

Comment:
I would really very much rather see `promote()` take `args` as a parameter, than use a global

Reply:
yeah...I'm always on the fence when I split up main() and then have to deal with args.  Invocation args to me are inherently global and I think they clutter up the function signature.  But I can acquiesce here, particularly since there aren't any other args.

Reply:
args passed to workers.